### PR TITLE
📝 updated hard mode validator onboarding documentation

### PR DESCRIPTION
**Pull request by ownyrd**
## Motivation

I adjusted the onboarding docs a little bit so that it is easier to follow.

### CODE REVIEW

File: ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md [hunk 1]
```diff
@@ -1,79 +1,118 @@
...
+    - [cargo or rust are not installed](#cargo-or-rust-are-not-installed)
+
+## Requirements
+
 - TWO unix hosts, one for Validator Node, and one for the Private Fullnode ("VFN").
 0L code targets Ubuntu 20.4
-- Recommended specs: 
-  - Validator: 250G harddrive, 8 core CPU, 16G RaM
+- Recommended specs:
+  - Validator: 250G harddrive, 8 core CPU, 16G RAM
```
Comment (codilion) [inline ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md:31]: Let's bump the hard drive requirement to 300GB SSD.

File: ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md [hunk 2]
```diff
@@ -343,93 +406,109 @@ diem-node --config ~/.0L/vfn.node.yaml  >> ~/.0L/logs/node.log 2>&1
...
 > ```txs create-validator -u http://[your-ip-address]```
-
-
+>
+> Currently you need 4 validators to vote for you (your onboading partner included), you can kindly ask in the [validator-onboarding discord channel](https://discord.com/channels/833074824447655976/925880684193677332).
 
 ## 7. Restart validator node in `validator` mode
 
-Once the network is in sync and sufficient mining has been done (20 proofs per epoch/day), you are eligible to enter the 
+Once the network is in sync, sufficient mining has been done (20 proofs per epoch/day) and you received the 4 vouches, you are eligible to enter the
```
Comment (codilion) [inline ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md:432]: Both validators and miners only need 8 proofs per epoch to be eligible to be in the active set or receive rewards.

**Review by codilion — CHANGES_REQUESTED** [review]
Great job.

**Review by 0xzoz — APPROVED** [review]
LGTM! Great work with this. Very good editing job.

File: ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md (hunk 1 above)
Comment (ownyrd) [reply ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md]: modified

File: ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md (hunk 2 above)
Comment (ownyrd) [reply ol/documentation/node-ops/validators/validator_onboarding_hard_mode.md]: okay, changed it

**Comment by ownyrd** [comment]
@codilion and @0xzoz thanks for the quick review. I'm very happy to hear that things are looking good for you guys.

**Review by codilion — APPROVED** [review]

**Pull request merged**